Lib/tempfile.py

```diff
@@ -697,6 +699,15 @@ def encoding(self):
                 raise
             return self._TemporaryFileArgs['encoding']
 
+    @property
+    def errors(self):
+        try:
+            return self._file.errors
+        except AttributeError:
```

Comment:
Is it needed?

Reply:
Accessing `self._file.errors` might raise `AttributeError` because in this class `_file` is initially a `StringIO` or `BytesIO` instance. The workaround with catching `AttributeError` is maybe not optimal, but that's the way it's done for the other file attributes (like `encoding`, `mode`, ...).

Reply:
Yes, but StringIO has the errors attribute, and in the case of BytesIO the exception will be reraised.

Reply:
I wasn't even aware that StringIO has this attribute :). It does indeed work the same without the `try`.